test/typescript/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "strict": true,
     "baseUrl": "../..",
     "paths": {
-      "redux": ["types/index.d.ts"]
+      "redux": ["lib/redux.d.ts"]
```

Comment:
Is this needed at all? Where are we importing from the `redux` package rather than the source code? I could see someone dealing with an out of sync build and running tests that give a false positive/negative.

Reply:
In the tests, there are a number of imports directly from `"redux"`. I see no reason these can't be changed to relative paths though.

Reply:
See 17fa65a